js/preferences/Preferences.js

```diff
@@ -103,6 +126,33 @@ export function save_on_run_task_set(save_on_run_task) {
     return save();
 }
 
+/**
+ * Get the desired runner type.
+ * @returns {RunnerType}
+ */
+export function runner_get() {
```

Comment:
I think the naming convention wasn't overly clear in retrospect so later on I might change these all (ie the similar ones I did and your new ones) to use `prop_get_or_fallback()` or something. More verbose but better distinction than `prop_get()` vs `prop_project_get()` since they're both querying the project properties.